### PR TITLE
Add TargetFramework .NET Core 3.1

### PR DESCRIPTION
**Pull request by KiKoS0**
Update .csproj to the new format and add support for multiple
.Net framework versions and .Net Core.

### CODE REVIEW

**Comment by KiKoS0** [comment]
I guess appveyor is failing because of an outdated build worker image that doesn't support the newest .net core SDK.

File: WpfAnimatedGif/WpfAnimatedGif.csproj [hunk 1]
```diff
@@ -1,73 +1,8 @@
...
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{D129789C-3096-4D0B-8DD7-FE24A4DF4B21}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>WpfAnimatedGif</RootNamespace>
-    <AssemblyName>WpfAnimatedGif</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile>Client</TargetFrameworkProfile>
+    <TargetFrameworks>net35;net4;net45;net46;net472;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```
Comment (thomaslevesque) [inline WpfAnimatedGif/WpfAnimatedGif.csproj]: Is it actually necessary to include all those frameworks?
For instance, net40 would be enough to support every net4* framework, and a netcoreapp3.1 app can use a netcoreapp3.0 assembly.
I think keeping just net35, net40 and netcoreapp3.0 is enough

**Comment by thomaslevesque** [comment]
I made a few more changes:
- added reference assemblies for .NET 3.5; without this, it won't build with `dotnet build`
  - since the official reference assemblies package doesn't include 3.5, I had to use a community package
- migrated the demo project to .NET Core SDK and netcoreapp3.0
  - had to replace usage of Interaction.InputBox, which no longer exists in .NET Core 3.0
- updated the build script and told AppVeyor to use it

It builds now, but I need to do some work to improve CI (currently it only checks that the code builds, it doesn't generate the package)

File: WpfAnimatedGif/WpfAnimatedGif.csproj (hunk 1 above)
Comment (KiKoS0) [reply WpfAnimatedGif/WpfAnimatedGif.csproj]: No not really, i was just tinkering with the versions and ended up with that lol. So keeping net35, net40 and netcoreapp3.0 should be enough.

**Comment by KiKoS0** [comment]
> Thanks @KiKoS0! I've been meaning to do this for a while but never found the time and motivation, so I really appreciate it.
You're welcome, glad i could help.

**Comment by thomaslevesque** [comment]
LGTM, thanks @KiKoS0!

**Comment by thomaslevesque** [comment]
I'll publish a new release after I make a few changes to the CI process (hopefully today or tomorrow)

**Pull request merged**